### PR TITLE
Improve sensor bias initialization and stability

### PR DESCRIPTION
**Pull request by mvacanti**
Observed instability in sensor bias initialization where PX4 reported values of mag, accel, and gyro sensor input were infinite or Nan on startup. Hypothesize that the way that the Eigen vectors were being initialized created a memory issue. Was unable to find an alternate using Eigen.

### CODE REVIEW

File: include/sensor_imu_plugin.h [hunk 1]
```diff
@@ -74,8 +74,6 @@ class SensorImuPlugin : public SensorPlugin {
   /// Norm of the gravitational acceleration [m/s^2]
   double gravity_magnitude;
 
-  Eigen::Vector3d _gyroscope_bias;
-  Eigen::Vector3d _accelerometer_bias;
```
Comment (Jaeyoung-Lim) [inline include/sensor_imu_plugin.h:78]: In there a reason that we want to remove this as a member of the imu plugin class? why not initialize it in the constructor?

**Comment by mvacanti** [comment]
I do not have a great technical answer but here are my observations and hypothesis:

- The problem inconsistently manifests itself. 
- Building the quadrotor or rascal does not present issues.
- **building the hex produces the following:**

Screenshot from 2020-09-29 04-59-16

- **Printing the _gyroscope_bias:**

Screenshot from 2020-09-29 05-03-34

- After making changes to the sensor_imu_plugin the problem moved to the sensor_mag_plugin...

I am not a C++ expert but what I hypothesize is happening is that the initialization of the Eigen vector is the root of the problem - it is expected to be initialized as zeros but occasionally does not (memory address?). 

[http://eigen.tuxfamily.org/dox/group__TutorialAdvancedInitialization.html](url)

When I attempt to initialize a std::vector in the constructor it is not possible to declare it with zeros and therefore the program never publishes the sensor to PX4 as the calculation of the bias references itself on the first iteration:

Screenshot from 2020-09-29 05-28-12

File: include/sensor_imu_plugin.h (hunk 1 above)
Comment (mvacanti) [reply include/sensor_imu_plugin.h:78]: Responded in the wrong location, see comment below:
[https://github.com/PX4/px4-jsbsim-bridge/pull/4#issuecomment-700591201](url)

**Comment by Jaeyoung-Lim** [comment]
@mvacanti I tried addressing the initialization issues in https://github.com/Auterion/px4-jsbsim-bridge/pull/14

Could you may be give it a try?

**Comment by mvacanti** [comment]
@Jaeyoung-Lim Thanks! This checks out on my end with all three airframes and as you are likely aware also fixes the GPS publication issue [mvacanti#6](https://github.com/PX4/px4-jsbsim-bridge/pull/6). Closed by [Auterion#14](https://github.com/Auterion/px4-jsbsim-bridge/pull/14).

**Pull request closed without merging**